constellations/orion/src/controllers/classes_by_semester_controller_test.go

```diff
@@ -0,0 +1,80 @@
+package controllers_test
```

Comment:
Make your test & regular controller file names consistent.
The test is called `classes_by_semester_controller_test.go` but the regular controller is: `classes_by_semesterS_controller.go`